contracts/mocks/WhitelistMock.sol

```diff
@@ -7,6 +7,7 @@ contract WhitelistMock is Whitelist {
 
   function onlyWhitelistedCanDoThis()
     onlyWhitelisted
+    isWhitelisted(msg.sender)
```

Comment:
Yup! Was being lazy and tested both codepaths at the same time. Honestly I'm just going to remove isWhitelisted and stick wtih onlyWhitelisted(address) because that's better anyway